examples/gatekeeper/managed-resource/samples/invalid_ownere_ref_disallowed.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: s3.aws.crossplane.io/v1beta1
+kind: Bucket
+metadata:
+  annotations:
+    crossplane.io/composition-resource-name: s3-bucket
+    crossplane.io/external-create-pending: "2022-09-29T16:36:09Z"
+    crossplane.io/external-create-succeeded: "2022-09-29T16:36:10Z"
+    crossplane.io/external-name: standard-object-storage-9mfmk-xgqv8
+  creationTimestamp: "2022-09-29T16:36:08Z"
```

Comment:
we can probably get rid of these extra post-submission labels to make it a bit cleaner. not a big deal. just for better readability